### PR TITLE
gpa 0.9.9 (new formula)

### PR DESCRIPTION
**Pull request by PhilipMay**

### CODE REVIEW

File: Library/Formula/gpa.rb [hunk 1]
```diff
@@ -0,0 +1,21 @@
+class Gpa < Formula
+  desc "Graphical user interface for the GnuPG"
+  homepage "https://www.gnupg.org/related_software/gpa/"
+  url "ftp://ftp.gnupg.org/gcrypt/gpa/gpa-0.9.9.tar.bz2"
```
Comment (DomT4) [inline Library/Formula/gpa.rb]: Please use the `https` version of this URL instead, Thanks! Upstream now support full TLS downloads, which is a nice bonus.

Comment (PhilipMay) [inline Library/Formula/gpa.rb]: Hi DomT4, there is no https Version of this download on gnupg.org. There is only this ftp version.

Comment (DomT4) [inline Library/Formula/gpa.rb]: `https://gnupg.org/ftp/gcrypt/gpa/gpa-0.9.9.tar.bz2`

**Comment by PhilipMay** [comment]
Thanks for the https URL DomT4. I changed it and all CI stuff is green.

**Comment by bfontaine** [comment]
Thank you for your contribution to Homebrew @PhilipMay; we appreciate it! :tada:

**Pull request closed without merging**